uniflow/node.py

```diff
@@ -1,5 +1,4 @@
-""" Node module for uniflow."""
-
```

Comment:
add a space here. Also. Update to the latest isort by running `pip3 install isort --upgrade` in your cli.

Reply:
due to your format error https://github.com/CambioML/uniflow/actions/runs/7683765586/job/20948001884?pr=133#step:6:56

Reply:
due to https://github.com/CambioML/uniflow/actions/runs/7683765586/job/20948001884?pr=133#step:6:56 format error